apps/dapp/src/services/AnalyticsService.ts

```diff
@@ -0,0 +1,35 @@
+import env from 'constants/env';
+import posthog from 'posthog-js';
+import { isDevelopmentEnv } from 'utils/helpers';
+
+export class AnalyticsService {
```

Comment:
Why don't we export the instance directly? Currently, every time you want to use it, you have to import the service, then get the instance. It would be simpler to do something like :

```
 import { analytics } from 'services/AnalyticsService';
 
 analytics.init()
 analytics.captureEvent()

```

Reply:
Yes I'll simplify it.

Reply:
https://github.com/TempleDAO/temple/pull/486/commits/59ec0703e6bc95876708be100bdcdef4e74ceb5d